data/wacom.metainfo.xml.in

```diff
@@ -31,6 +31,15 @@
   <update_contact>contact_at_elementary.io</update_contact>
 
   <releases>
+    <release version="8.0.1" date="2024-12-17" urgency="medium">
+      <description>
+        <p>Updated translations</p>
+      </description>
+      <issues>
+        <issue url="https://github.com/elementary/switchboard-plug-wacom/issues/69">A fix for a crush of Settings that occured when stylus is not detected by libwacom.</issue>
```

Comment:
```suggestion
        <issue url="https://github.com/elementary/switchboard-plug-wacom/issues/69">Clicking on the wacom plug crushes The switchboard</issue>
```

Reply:
Updated issue title and therefore the `<issue>` tag contents.